tests/test_resolve.py

```diff
@@ -64,16 +65,36 @@ def test_resolve_with_version(data_archive, cli_runner):
             pk = conflict_id.split(":", 2)[2]
             pk_order += [pk]
 
-            r = cli_runner.invoke(
-                ["resolve", conflict_id, f"--with={next(resolutions)}"]
-            )
+            resolution = next(resolutions)
+            r = cli_runner.invoke(["resolve", conflict_id, f"--with={resolution}"])
             assert r.exit_code == 0, r.stderr
             conflict_ids = get_conflict_ids(cli_runner)
             assert len(conflict_ids) == num_conflicts - 1
 
             resolved_keys = MergedIndex.read_from_repo(repo).resolves.keys()
             ck_order += [k for k in resolved_keys if k not in ck_order]
 
+            if resolution in ("ancestor", "delete"):
+                with repo.working_copy.tabular.session() as sess:
+                    count = sess.scalar(
+                        f"""SELECT COUNT(*) FROM nz_waca_adjustments WHERE id = {pk};"""
```

Comment:
IMO we should use proper parameterisation, even in tests - otherwise some less experienced programmer will copy from here and paste somewhere that matters

Reply:
Done

Reply:
Done